WebRoot/js/zimbraAdmin/accounts/model/ZaAccount.js

```diff
@@ -292,7 +292,7 @@ ZaAccount.A_zimbraAuthLdapExternalDn = "zimbraAuthLdapExternalDn";
 ZaAccount.A_zimbraFreebusyExchangeUserOrg = "zimbraFreebusyExchangeUserOrg" ;
 ZaAccount.A_zimbraFeatureManageSMIMECertificateEnabled = "zimbraFeatureManageSMIMECertificateEnabled";
 ZaAccount.A_zimbraFeatureSMIMEEnabled = "zimbraFeatureSMIMEEnabled";
-ZaAccount.A_zimbraFeatureZulipChatEnabled = "zimbraFeatureZulipChatEnabled";
+// ZaAccount.A_zimbraFeatureZulipChatEnabled = "zimbraFeatureZulipChatEnabled";
```

Comment:
Since we are not using this, we can remove commented code. And whenever we need it we can always get it from history :)